<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideLeft</enum>
    </property>
    <widget class="QWidget" name="process_tab">
     <attribute name="title">
      <string>进程</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="cpu_tab">
     <attribute name="title">
      <string>CPU</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="memory_tab">
     <attribute name="title">
      <string>内存</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="network_tab">
     <attribute name="title">
      <string>网络</string>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
